### ISSUE TITLE
Unable to build on Xcode 11.3.1

### ISSUE THREAD

**Issue by hassan-rafique** [tanersener/mobile-ffmpeg#419]
**Description**
I am unable to integrate this into my project. It gives a compilation error. I added using pods:

`pod 'mobile-ffmpeg-full', '~> 4.3.2'`

**Expected behavior**
It should run smoothly without any issues

**Current behavior**
Gives compilation error.

**Screenshots**
<img width="1137" alt="Screenshot 2020-05-11 at 12 05 53 AM" src="https://user-images.githubusercontent.com/13598760/81508155-d67ae600-931b-11ea-85bd-7e290186e003.png">

**Environment**
 - Platform: iOS

**Comment by NickBusey** [comment]
I am blocked by this as well, I would love to help try and debug but don't know quite where to start. Any hints @tanersener ? Thanks!

**Comment by tanersener** [comment]
- Did you try deleting `DerivedData` folder, also `Pods` folder?
- Can you try running test-application provided under `ios/test-app/pods-with-tooltips` folder?

**Comment by NickBusey** [comment]
I don't see a `DerivedData` folder. I deleted the `Pods` folder, then re-ran `pod install`, same result.

Trying to run the test app (I cloned the repo, inside the test app folder ran `pod install`, opened the xcworkspace file, and hit run) I get a similar looking error:

```
Ld /Users/nick/Library/Developer/Xcode/DerivedData/MobileFFmpegTest-ahvmiuzogbreboeahicnstvrzarb/Build/Products/Debug-iphonesimulator/MobileFFmpegTest.app/MobileFFmpegTest normal x86_64 (in target 'MobileFFmpegTest' from project 'MobileFFmpegTest')
    cd /Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -target x86_64-apple-ios11.0-simulator -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk -L/Users/nick/Library/Developer/Xcode/DerivedData/MobileFFmpegTest-ahvmiuzogbreboeahicnstvrzarb/Build/Products/Debug-iphonesimulator -F/Users/nick/Library/Developer/Xcode/DerivedData/MobileFFmpegTest-ahvmiuzogbreboeahicnstvrzarb/Build/Products/Debug-iphonesimulator -F/Users/nick/Library/Developer/Xcode/DerivedData/MobileFFmpegTest-ahvmiuzogbreboeahicnstvrzarb/Build/Products/Debug-iphonesimulator/NMEasyTipView -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/expat.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/expat.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/expat.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/fontconfig.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/fontconfig.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/fontconfig.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/freetype.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/freetype.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/freetype.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/fribidi.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/fribidi.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/fribidi.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/giflib.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/giflib.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/giflib.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/gmp.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/gmp.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/gmp.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/gnutls.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/gnutls.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/gnutls.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/jpeg.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/jpeg.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/jpeg.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/kvazaar.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/kvazaar.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/kvazaar.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/lame.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/lame.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/lame.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libaom.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libaom.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libaom.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libass.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libass.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libass.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavcodec.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavcodec.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavcodec.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavdevice.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavdevice.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavdevice.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavfilter.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavfilter.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavfilter.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavformat.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavformat.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavformat.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavutil.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavutil.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libavutil.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libhogweed.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libhogweed.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libhogweed.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libilbc.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libilbc.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libilbc.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libnettle.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libnettle.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libnettle.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libogg.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libogg.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libogg.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libopencore-amrnb.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libopencore-amrnb.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libopencore-amrnb.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libopencore-amrwb.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libopencore-amrwb.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libopencore-amrwb.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libpng.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libpng.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libpng.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libsndfile.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libsndfile.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libsndfile.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libswresample.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libswresample.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libswresample.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libswscale.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libswscale.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libswscale.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libtheora.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libtheora.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libtheora.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libtheoradec.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libtheoradec.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libtheoradec.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libtheoraenc.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libtheoraenc.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libtheoraenc.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvorbis.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvorbis.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvorbis.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvorbisenc.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvorbisenc.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvorbisenc.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvorbisfile.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvorbisfile.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvorbisfile.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvpx.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvpx.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libvpx.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libwebp.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libwebp.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libwebp.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libwebpdemux.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libwebpdemux.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libwebpdemux.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libwebpmux.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libwebpmux.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libwebpmux.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libxml2.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libxml2.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/libxml2.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/mobileffmpeg.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/mobileffmpeg.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/mobileffmpeg.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/opus.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/opus.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/opus.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/shine.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/shine.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/shine.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/snappy.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/snappy.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/snappy.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/soxr.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/soxr.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/soxr.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/speex.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/speex.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/speex.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/tiff.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/tiff.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/tiff.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/twolame.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/twolame.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/twolame.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/wavpack.xcframework/ios-arm64 -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/wavpack.xcframework/ios-x86_64-maccatalyst -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips/Pods/mobile-ffmpeg-full/wavpack.xcframework/ios-x86_64-simulator -F/Users/nick/Code/mobile-ffmpeg/ios/test-app/pods-with-tooltips -filelist /Users/nick/Library/Developer/Xcode/DerivedData/MobileFFmpegTest-ahvmiuzogbreboeahicnstvrzarb/Build/Intermediates.noindex/MobileFFmpegTest.build/Debug-iphonesimulator/MobileFFmpegTest.build/Objects-normal/x86_64/MobileFFmpegTest.LinkFileList -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/nick/Library/Developer/Xcode/DerivedData/MobileFFmpegTest-ahvmiuzogbreboeahicnstvrzarb/Build/Intermediates.noindex/MobileFFmpegTest.build/Debug-iphonesimulator/MobileFFmpegTest.build/Objects-normal/x86_64/MobileFFmpegTest_lto.o -Xlinker -export_dynamic -Xlinker -no_deduplicate -Xlinker -objc_abi_version -Xlinker 2 -fobjc-arc -fobjc-link-runtime -lbz2 -lc++ -liconv -lz -framework AVFoundation -framework AudioToolbox -framework CoreMedia -framework NMEasyTipView -framework VideoToolbox -framework expat -framework fontconfig -framework freetype -framework fribidi -framework giflib -framework gmp -framework gnutls -framework jpeg -framework kvazaar -framework lame -framework libaom -framework libass -framework libavcodec -framework libavdevice -framework libavfilter -framework libavformat -framework libavutil -framework libhogweed -framework libilbc -framework libnettle -framework libogg -framework libopencore-amrnb -framework libopencore-amrwb -framework libpng -framework libsndfile -framework libswresample -framework libswscale -framework libtheora -framework libtheoradec -framework libtheoraenc -framework libvorbis -framework libvorbisenc -framework libvorbisfile -framework libvpx -framework libwebp -framework libwebpdemux -framework libwebpmux -framework libxml2 -framework mobileffmpeg -framework opus -framework shine -framework snappy -framework soxr -framework speex -framework tiff -framework twolame -framework wavpack -Xlinker -sectcreate -Xlinker __TEXT -Xlinker __entitlements -Xlinker /Users/nick/Library/Developer/Xcode/DerivedData/MobileFFmpegTest-ahvmiuzogbreboeahicnstvrzarb/Build/Intermediates.noindex/MobileFFmpegTest.build/Debug-iphonesimulator/MobileFFmpegTest.build/MobileFFmpegTest.app-Simulated.xcent -framework Pods_MobileFFmpegTest -Xlinker -dependency_info -Xlinker /Users/nick/Library/Developer/Xcode/DerivedData/MobileFFmpegTest-ahvmiuzogbreboeahicnstvrzarb/Build/Intermediates.noindex/MobileFFmpegTest.build/Debug-iphonesimulator/MobileFFmpegTest.build/Objects-normal/x86_64/MobileFFmpegTest_dependency_info.dat -o /Users/nick/Library/Developer/Xcode/DerivedData/MobileFFmpegTest-ahvmiuzogbreboeahicnstvrzarb/Build/Products/Debug-iphonesimulator/MobileFFmpegTest.app/MobileFFmpegTest

ld: warning: could not create compact unwind for _ass_encode_frame: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _dnxhd_encode_picture: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _fits_decode_frame: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _decode_frame: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _g723_1_encode_frame: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _encode_picture_ls: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _svq1_encode_frame: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _xface_encode_frame: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _ff_init_vlc_sparse: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _ac3_eac3_probe: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _filter_frame: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _ff_celt_quant_bands: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _ff_cfhd_init_vlcs: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _fits_read_packet: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _ff_htmlmarkup_to_ass: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _ff_init_2d_vlc_rl: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _redspark_read_header: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _ff_rl_init_vlc: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _ff_rtsp_setup_output_streams: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _rtsp_write_header: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _rtsp_write_packet: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _udp_open: stack subq instruction is too different from dwarf stack size
ld: warning: could not create compact unwind for _ff_ape_parse_tag: stack subq instruction is too different from dwarf stack size
Undefined symbols for architecture x86_64:
  "___darwin_check_fd_set_overflow", referenced from:
      _transcode in mobileffmpeg(libmobileffmpeg_la-fftools_ffmpeg.o)
      _xmlNanoFTPCheckResponse in libxml2(nanoftp.o)
      _xmlNanoFTPCloseConnection in libxml2(nanoftp.o)
      _xmlNanoFTPList in libxml2(nanoftp.o)
      _xmlNanoFTPGet in libxml2(nanoftp.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by hassan-rafique** [comment]
@tanersener FYI: Test application provided in ios/test-app/pods-with-tooltips did not work for me as well.

**Comment by tanersener** [comment]
Can you share cocoapods version you have and Podfile.lock of possible?

**Comment by NickBusey** [comment]
1.9.1

```
PODS:
  - mobile-ffmpeg-full (4.3.2)
  - NMEasyTipView (1.2)

DEPENDENCIES:
  - mobile-ffmpeg-full (= 4.3.2)
  - NMEasyTipView (~> 1.2)

SPEC REPOS:
  trunk:
    - mobile-ffmpeg-full
    - NMEasyTipView

SPEC CHECKSUMS:
  mobile-ffmpeg-full: 5ff7bf33f70c7c89bba663e213f0d5a1ba8ddb84
  NMEasyTipView: 04948bb8ee01b22957095e1427be972115b57d02

PODFILE CHECKSUM: a484053f99debaa5b762bb36b460e762027d8d45

COCOAPODS: 1.9.1
```

**Comment by tanersener** [comment]
Thanks. Your `Podfile.lock` file looks fine. `___darwin_check_fd_set_overflow` symbol error is the main issue here. It is specific to Xcode`11.3.1` and `MobileFFmpeg v4.3.2`.

`MobileFFmpeg v4.3.2` was built using the newer `Xcode 11.4` and I guess there is an incompatibility between these two Xcode versions. That's why you this issue. By the way, `v4.3.2` works fine with `Xcode 11.4.x`.

The same issue is reported in other projects as well. There are also some reports that it is an Apple bug. It is hard to tell if Apple will do something about it.

You can overcome this failure by:
- Upgrading to `Xcode 11.4.x`or
- Using a `MobileFFmpeg` version other than `v4.3.2`

**Comment by NickBusey** [comment]
I can confirm upgrading Xcode to 11.4.x fixes this problem.

**Comment by tanersener** [comment]
Added a note about this issue on `v4.3.2` release notes.

**Comment by jyounus** [comment]
I'm guessing there are no plans to make it work for anything below Xcode 11.4? I'm asking as I haven't updated to Catalina yet (as I really don't want to), and 11.4 doesn't work on Mojave.

**Comment by tanersener** [comment]
I'm not sure whether this project needs to do anything about it or not. I believe that incompatibility between Xcode versions is causing this.

Btw, why don't you use a MobileFFmpeg version other than `v4.3.2`? They're working fine with `Xcode 11.3.x`.

**Comment by aaronkaz** [comment]
Apparently the same issue is repeated on latest Xcode v12.0

**Comment by tanersener** [comment]
I don't have any issues with `Xcode 12.0`. Please submit a new issue if you have problems about it.

**Comment by dancixx** [comment]
Same issue on Xcode 12.0. :(

```
in /Users/tamaskoller/Desktop/connec-fitness-rn/ios/Pods/mobile-ffmpeg-https/libavcodec.framework/libavcodec(aacencdsp.o), building for iOS Simulator, but linking in object file built for free standing, file '/Users/tamaskoller/Desktop/connec-fitness-rn/ios/Pods/mobile-ffmpeg-https/libavcodec.framework/libavcodec' for architecture x86_64
```
